eng/pipelines/aggregate-reports.yml

```diff
@@ -55,7 +55,7 @@ jobs:
       displayName: 'Generate aggregate code coverage report'
```

Comment:
Update title to say code coverage and dependencies

Reply:
Updating to `Generate aggregate code coverage and dependency report`